### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by BreakingPitt**

### CODE REVIEW

**Review by egarbi — COMMENTED** [review]
Hey @BreakingPitt , if we are going to split files as recommended by terraform, the right name for variables file is `variables.tf` Could you please change it? Gracias

**Comment by BreakingPitt** [comment]
Sure... Sorry!!!!

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+variable "dedicated_master" {
+  default     = false
+  description = "Indicates whether our cluster have dedicated master nodes enabled."
+  type        = "string"
+}
+
+variable "encryption_enabled" {
+  default     = "no"
```
Comment (egarbi) [inline variables.tf]: default value here need to be "false", "no" its not valid according this:
https://www.terraform.io/docs/providers/aws/r/elasticsearch_domain.html#enabled

Comment (BreakingPitt) [reply variables.tf]: Sorry... I created this in a initial state and then changed its values but seems that I forgot to change this.

P.D - > Gracias majo!

**Pull request merged**